test/localhub/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.9"
+services:
+  tests:
+    image: "${DOCKER_REPOSITORY}/localhub-test:${DOCKER_TAG}"
+    depends_on:
+      hubjs:
+        condition: service_healthy
+    environment:
+      - LOCALHUB_ADDR=http://hubjs:8080/graphql
+      - GRPC_SECRET_STORAGE_BACKEND_ADDR=http://backend-storage:50051
+  hubjs:
+    image: "${DOCKER_REPOSITORY}/hub-js:${DOCKER_TAG}"
+    restart: on-failure
+    ports:
+      - "8080:8080"
+    environment:
+      - APP_NEO4J_ENDPOINT=bolt://neo4j:7687
+      - NEO4JLABS_PLUGINS=["apoc"]
+      - APP_NEO4J_PASSWORD=okon
+      - APP_HUB_MODE=local
+      - APP_GRAPH_QL_ADDR=:8080
+    depends_on:
+      neo4j:
+        condition: service_healthy
+    healthcheck:
+      test: netstat -ltn | grep -c 8080
```

Comment:
We have dedicated healthz endpoints for our services, please use them - both for Local Hub, and Test-storage-backend 🙂  See the charts for details

For example, for hub js:
```
    httpGet:
      path: /healthz
      port: 8080
```